app/services/container_service_mixin.rb

```diff
@@ -63,16 +64,16 @@ def utilization_data(used_cpu, total_cpu, used_mem, total_mem)
     if used_cpu.any?
       {
         :cpu => {
-          :used  => used_cpu.values.last.round,
-          :total => total_cpu.values.last.round,
+          :used  => (used_cpu.values.last || 0).round(DISPLAY_PRECISION),
+          :total => (total_cpu.values.last || 0).round,
           :xData => used_cpu.keys,
-          :yData => used_cpu.values.map(&:round)
+          :yData => used_cpu.values.map { |m| m.round(GRAPH_USAGE_PRECISION) }
```

Comment:
This feels like it can be more generic. This is the precision we would want for all graph values so maybe this can be GRAPH_PRECISION?

Reply:
:+1: